types/messages.go

```diff
@@ -125,3 +127,62 @@ func (msg *SSVMessage) Encode() ([]byte, error) {
 func (msg *SSVMessage) Decode(data []byte) error {
 	return msg.UnmarshalSSZ(data)
 }
+
+// SSVMessage is the main message passed within the SSV network. It encapsulates the SSVMessage structure and a signature
+type SignedSSVMessage struct {
+	OperatorID OperatorID
+	Signature  []byte `ssz-max:"512"`     // Current signature max size allow keys up to 512*8 = 4096 bits
```

Comment:
maybe it is just nice to comment that this is created by the operator's private key